validation/pioneer-study/esper-study-telemetry.1.sample.json

```diff
@@ -0,0 +1,43 @@
+{
+  "defaultSearchEngine": "google",
+  "locale": "en-US",
+  "os": "Darwin",
+  "normalizedChannel": "nightly",
```

Comment:
The "normalized" channel is not information that is available on the client, as far as I know. One should simply collect the "app update channel" on the client and the server will normalize it as usual.

Reply:
Thanks, fixed